recipes/DogImagesReactHooks/README.md

```diff
@@ -0,0 +1,14 @@
+# DogImagesReactHooks
+
+A React port of the (now defunct) ["HTTP - Cat GIFs" Elm Example](https://elm-lang.org/examples/cat-gifs), refactored to use the [Dog API](https://dog.ceo/dog-api/).
```

Comment:
Maybe this shouldn't even reference the Elm example anymore, since the recipes no longer use cat images and that link now redirects to https://elm-lang.org/examples/quotes